program/src/error.rs

```diff
@@ -88,6 +88,10 @@ pub enum StakeError {
     /// 19 - Invalid slash amount
     #[error("Invalid slash amount")]
     InvalidSlashAmount,
+
+    /// 20 - Total stake amount exceeds limit
+    #[error("Total stake amount exceeds limit")]
+    TotalStakeAmountExceedsLimit,
```

Comment:
nit: Maybe this can be clearer, like "Total PAL stake amount exceeds 1.3 * SOL stake amount", with name `TotalPalStakeAmountExceedsLimit`. What do you think?

Reply:
It definitely looks better, but I wonder:
1. We haven't made direct references to PAL token in the program so far and technically you could use it with other tokens.
2. the `1.3` multiplier might end up not being fixed, so I wonder if it is a good idea to have it on an error message.

A bit related to this, I was going to ask whether we should consider renaming the `total_staked_pal_amount` field to something more generic or not – like `total_staked_token_amount` – similar to how the name is on the `Config`.

https://github.com/paladin-bladesmith/stake-program/blob/main/program/src/state/validator_stake.rs#L36

Reply:
Ah gotcha, that's a good point. Feel free to rename if you think it's clearer. In that case though, maybe we can call the error `TotalStakeAmountExceedsSolLimit` :smile: